Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -1081,6 +1081,9 @@ public function testShardManager(): void
         $this->assertEquals($managerClass, $bazManagerDef->getClass());
     }
 
+    // Disabled to prevent changing the comment below to a single-line annotation
+    // phpcs:disable SlevomatCodingStandard.Commenting.RequireOneLineDocComment.MultiLineDocComment
+
```

Comment:
PHPUnit does not seem to understand one-line annotations, so we need to disable the sniff to avoid phpcbf changing this back.

Reply:
can we instead disable that sniff for the whole Tests folder through the config, to avoid introducing similar issues in the future when using PHPUnit annotations on other tests ?

Reply:
I would rather bug was fixed in phpunit

Reply:
long term, indeed.

Reply:
Fixed in https://github.com/sebastianbergmann/phpunit/pull/4693